yt_dlp/networking/_urllib.py

```diff
@@ -50,6 +50,10 @@
     SUPPORTED_ENCODINGS.append('br')
     CONTENT_DECODE_ERRORS.append(brotli.error)
 
+if zstandard:
```

Comment:
more minutia: safer to do this instead of truthiness check?
```suggestion
if zstandard is not None:
```